core/contextmenu_items.js

```diff
@@ -0,0 +1,272 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Registers default context menu items.
+ * @author maribethb@google.com (Maribeth Bottorff)
+ */
+'use strict';
+
+/**
+ * @name Blockly.ContextMenuItems
+ * @namespace
+ */
+goog.provide('Blockly.ContextMenuItems');
+
+goog.requireType('Blockly.BlockSvg');
+
+/** Option to undo previous action. */
+var registerUndo = function() {
```

Comment:
Should we place these methods on the `Blockly.ContextMenuItems` object so they can be tested / potentially re-used by others?

Reply:
OK, added all the register* functions to the class but not the helper functions.